vast/integration/vast_integration_suite.yaml

```diff
@@ -785,3 +785,27 @@ tests:
     tags: [pipelines]
     steps:
       - command: exec 'from file @./data/json/record-in-list.json read json | write json to stdout'
+
+  Measure Events:
+    tags: [pipelines]
+    steps:
+      - command: exec 'read json | measure | drop timestamp | write json'
+        input: data/json/files.log.json.gz
+      - command: exec 'read json | measure | measure | drop timestamp | write json'
+        input: data/json/files.log.json.gz
+      - command: exec 'read json | measure --real-time | drop timestamp | write json'
+        input: data/json/files.log.json.gz
+      - command: exec 'read json | measure --real-time | measure | drop timestamp | write json'
+        input: data/json/files.log.json.gz
+
+  Measure Bytes:
+    tags: [pipelines]
+    steps:
+      - command: exec 'load stdin | measure | drop timestamp | write json'
+        input: data/json/conn.log.json.gz
+      - command: exec 'load stdin | measure | measure | drop timestamp | write json'
```

Comment:
Higher-order metrics. 😜

Reply:
This actually allows for seeing whether `measure --real-time` works, because all entries in `measure --real-time | measure` must show a batch size of 1.